packages/nft-analytics/src/models/twap.ts

```diff
@@ -0,0 +1,36 @@
+import { AdapterError } from '@chainlink/ea-bootstrap'
+
+import { Provider } from '../api/constants'
+import { NFTPerpTwapsGetPayload } from '../api/nftperp/types'
+
+export interface ITwap {
+  price: string
+}
+
+const transformer = {
+  [Provider.NFTPERP]: (payload: NFTPerpTwapsGetPayload): ITwap => {
+    return {
```

Comment:
We can provide more information about the collection and price. At least provide the `twap` and `timestamp` properties cause the API either returns the data or indirectly provides it.

```typescript
    return {
      collection: {
        contractAddress: null,
        name: null, // TODO: put the slug or leave it as null
        symbol: null,
        imageUrl: null,
      },
      twap: {
        currency: todo, // TODO: confirm whether we do only request ETH NFTs or other chain ones, otherwise null
        amount: payload.price,
        units: 'wei',
      },
      network: null,
      timestamp: todo, // TODO: ISO8061 to timestamp,
    }
```